test/Microsoft.ML.TimeSeries.Tests/TimeSeriesDirectApi.cs

```diff
@@ -661,6 +660,99 @@ public void AnomalyDetectionWithSrCnn(bool loadDataFromFile)
             }
         }
 
+        [Theory, CombinatorialData]
+        public void TestSrCnnAnomalyDetectorWithSeasonalData(
+            [CombinatorialValues(SrCnnDeseasonalityMode.Stl, SrCnnDeseasonalityMode.Mean, SrCnnDeseasonalityMode.Median)] SrCnnDeseasonalityMode mode
+        )
+        {
+            var ml = new MLContext(1);
```

Comment:
>MLContext [](start = 25, length = 9)

The base class has an `MLContext` field.

Reply:
Could you point out the `MLContext` field? As I find in this class, other tests are creating `MLContext` by themselves.

---
In reply to: [443415149](https://github.com/dotnet/machinelearning/pull/5202#discussion_r443415149) [](ancestors = 443415149)

Reply:
Sorry, I was wrong. The `MLContext` is not in `BaseTestClass`, it is in `BaseTestBaseline`, which is derived from `BaseTestClass`.

---
In reply to: [443532513](https://github.com/dotnet/machinelearning/pull/5202#discussion_r443532513) [](ancestors = 443532513,443415149)